x-pack/platform/plugins/shared/fleet/server/services/cloud_connector.ts

```diff
@@ -34,6 +38,20 @@ export interface CloudConnectorServiceInterface {
     soClient: SavedObjectsClientContract,
     options?: CloudConnectorListOptions
   ): Promise<CloudConnectorResponse[]>;
+  getById(
+    soClient: SavedObjectsClientContract,
+    cloudConnectorId: string
+  ): Promise<CloudConnectorResponse>;
```

Comment:
`CloudConnectorResponse` should be renamed to just `CloudConnector` as it is the type for the cloud connector object, not an API response schema